--- sys/unix/Makefile.top.orig	2019-05-13 11:07:15.731247038 -0500
+++ sys/unix/Makefile.top	2019-05-13 11:10:12.118232737 -0500
@@ -27,7 +27,7 @@
 
 # Permissions - some places use setgid instead of setuid, for instance
 # See also the option "SECURE" in include/config.h
-#GAMEPERM = 04755
+GAMEPERM = 02755
 FILEPERM = 0644
 # VARFILEPERM = 0644
 EXEPERM  = 0755
@@ -49,7 +49,7 @@
 # per discussion in Install.X11 and Install.Qt
 #VARDATND = 
 # VARDATND = x11tiles NetHack.ad pet_mark.xbm pilemark.xbm
-# VARDATND = x11tiles NetHack.ad pet_mark.xbm pilemark.xbm rip.xpm
+VARDATND = x11tiles NetHack.ad pet_mark.xbm pilemark.xbm rip.xpm
 # for Atari/Gem
 # VARDATND = nh16.img title.img GEM_RSC.RSC rip.img
 # for BeOS
